data/anaconda.conf

```diff
@@ -76,6 +76,14 @@ enabled_repositories_from_treeinfo = addon optional variant
 # Enable installation from the closest mirror.
 enable_closest_mirror = True
 
+# Enable %packages --ignorebroken feature.
```

Comment:
This configuration option doesn't enable only the kickstart option, but also the DBus support, so this description is not correct. I would keep it more general.

Reply:
Reworded.